common/culture/innovations/wc_regional_innovations.txt

```diff
@@ -44,3 +51,106 @@ innovation_war_camels = {
 	flag = global_regional
 	flag = tribal_era_regional
 }
+###Tech
+##Tribal
+
+innovation_sea_faring = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = custom_sea_locked
+	icon = @sea_faring
+	
+	potential = {
+		NOT = {
+			this.culture_group = culture_group:vrykul_group
+		}
+	}
+	modifier = {
+		embarkation_cost_mult = -0.8
+		garrison_size = -0.2
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regional
+}
+innovation_kyparite_sap = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	region = custom_kyparite_regions
+	icon = @kyparite_sap
+
+	modifier = {
+		garrison_size = 0.05
+		skirmishers_damage_mult = 0.1
+		skirmishers_toughness_mult = 0.05
+		monthly_prestige_gain_mult = 0.05		
+	}
+
+	flag = global_regional
+	flag = tribal_era_regional
+}
+##Early Medieval
+innovation_frigid_warfare = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	region = custom_frigid_regions
+	icon = @taiga
+
+	modifier = {
+		taiga_advantage = 3
+		mountains_advantage = 2
+		taiga_cancel_negative_supply = yes
+		mountains_cancel_negative_supply = yes
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regional
+}
+innovation_arid_warfare = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	region = custom_arid_regions
+	icon = @desert
+
+	modifier = {
+		desert_advantage = 3
+		drylands_advantage = 2
+		desert_cancel_negative_supply = yes
+		drylands_cancel_negative_supply = yes
+	}
+
+	flag = global_regional
+	flag = early_medieval_era_regional
+}
+innovation_guerilla_warfare = {
```

Comment:
Gurubashi doesn't leave in jungles.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114572097-40b4ad00-9c88-11eb-9fb3-7757af141afc.png)

</details>